at.priv.toastfreeware.ConfClerk.appdata.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2019 Tobias Mueller <tobiasmue@gnome.org>-->
+<component type="desktop">
+  <id>cat.priv.toastfreeware.ConfClerk.desktop</id>
+  <translation type="qt">ConfClerk</translation>
+  <name>ConfClerk</name>
+  <summary>ConfClerk is an application written in ​Qt, which makes conference schedules available offline. It displays the conference schedule from various views, supports searches on various items (speaker, speech topic, location, etc.) and enables you to select favorite events and create your own schedule</summary>
+  <developer_name>confclerk developers</developer_name>
+  <description>
+   <p>
+    At the moment ConfClerk is able to import schedules in XML format created by the ​PentaBarf conference management system (or ​frab) used by ​FOSDEM, ​DebConf, ​Grazer Linuxtage, the ​CCC congresses, ​FrOSCon, and ​many others.
+   </p>
+   <p>
+    ConfClerk is targetted at mobile devices like the Nokia N810 and N900 but works on any system running ​Qt.
+   </p>
+  </description>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0</project_license>
+  <url type="bugtracker">http://www.toastfreeware.priv.at/confclerk/report</url>
+  <url type="help">http://www.toastfreeware.priv.at/confclerk/browser/README</url>
+  <url type="homepage">http://www.toastfreeware.priv.at/confclerk/browser/</url>
+  <screenshots>
+    <screenshot type="default">
+      <image>http://www.toastfreeware.priv.at/confclerk/raw-attachment/wiki/WikiStart/Screenshot-20130128-180409.png</image>
+    </screenshot>
+    <screenshot>
+      <image>http://www.toastfreeware.priv.at/confclerk/raw-attachment/wiki/WikiStart/2013-01-28-175815_517x502_scrot.png</image>
+    </screenshot>
+  </screenshots>
+  <launchable type="desktop-id">@app-id@.desktop</launchable>
```

Comment:
```suggestion
  <launchable type="desktop-id">cat.priv.toastfreeware.ConfClerk.desktop</launchable>
```

Reply:
umm. so is `@app-id@` not a thing? I thought something in the process replaces it with the appropriate value. And it does indeed work for me, locally, to launch the app through GNOME Shell.
(and it's not "cat", but "at")

Reply:
the @app-id@ was never a thing as far as I know.  A lot of applications uses that because the appdata file is pre-processed by the build system to replace the `@app-id@` with its value

Reply:
In that case the id is wrong in the appdata file as it uses cat .